.github/dependabot.yml

```diff
@@ -0,0 +1,31 @@
+version: 2
+updates:
+  # Enable version updates for npm
+  - package-ecosystem: "npm"
```

Comment:
Can we group either all deps, or production + development deps so that there are fewer total PRs?  Our CI is a nightmare, and opening 1 PR per dependency seems like it will not be adopted by the team.

Reply:
https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#groups--